Pax.cs

```diff
@@ -178,7 +201,7 @@ public class Frontend {
           {
             Console.ForegroundColor = ConsoleColor.Red;
             Console.WriteLine("No match for '" + i.interface_name + "'");
-            return -1;
+            Environment.Exit(-1);
```

Comment:
@niksu: no, it causes the program to [terminate](https://msdn.microsoft.com/en-us/library/system.environment.exit(v=vs.110).aspx). Doesn't quite have the same semantics as return -1 though.

The alternative would be to return an error code from the method and check if it's non-zero at the call site.